libmscore/beam.cpp

```diff
@@ -1586,6 +1595,11 @@ void Beam::layout2(QList<ChordRest*>crl, SpannerSegmentType, int frag)
                               c->setUp(nup);
                               // guess was wrong, have to relayout
                               score()->layoutChords1(c->segment(), c->staffIdx());
+                              c->layout();
```

Comment:
Seems to be, yes.  Because we are in the "else" clause of the test for whether the beam is user modified, it's a given we are not in edit mode.  Of course, I'm still kind of guessing at what the exact cause of the crash was in the user-modified case - the crash happens during painting, while walking the bsp tree and coming across a bad element.  But ledger lines seems to likely culprit, since they re-genered on layout.